cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -1024,6 +1024,14 @@ macro(prepare_fetchcontent)
   # We should remove it once we have updated the dependencies:
   # https://github.com/apache/arrow/issues/45985
   set(CMAKE_POLICY_VERSION_MINIMUM 3.5)
+  # Use "NEW" for CMP0077 by default.
+  #
+  # https://cmake.org/cmake/help/latest/policy/CMP0077.html
+  #
+  # option() honors normal variables.
+  set(CMAKE_POLICY_DEFAULT_CMP0077
```

Comment:
So we don't need `CACHE ... FORCE` because of this?

Reply:
Right.